backend/bot/index.js

```diff
@@ -59,19 +59,28 @@ const postMatchResult = async (match) => {
 
   const isComedyDuo = winningPlayers.length == 2 && winningTeam.every(player => player.name === 'Pepa' || player.name === 'Tonda')
 
-  let prefix = '';
+  const messageParts = []
+
   if (isComedyDuo) {
-    prefix = ':tondab: :pepadab: ';
-  } else if (ratingChange <= 10) {
-    prefix = 'Easy. ';
-  } else if (ratingChange >= 20) {
-    prefix = 'HOLY SHIT! ';
+    messageParts.push(':tondab: :pepadab:')
```

Comment:
i know it's outside of the scope of the PR, bt yo touched the file, so I'm looking at you! 
now that we're opensourcing this project, could you please add a comment in the sources that explains the comedy duo?

Reply:
theoretically, you could have the messageParts built like you do and then do somthing like

applyComedyDuoMiddleware(messageParts, match) to make a pseudo chain-or responsibility thing. You don't need to do this if you don't want to though.

Reply:
I can add it but tell me what the comment should be because I have no idea how to explain that. :D Or just add the comment yourself then.

Sorry, I don't have the time to even think about the second part. :D

Reply:
"Comedy duo refers of two salsitistas known for their distinctive playing style" :D

Reply:
I will create a separate PR to generalize the functionality. See Slack.

Reply:
I have the code with the generalized "comedy duo" functionality ready but it's dependent on the code from here. Will create a new PR once this one is merged.